### PR TITLE
Polish gke-deploy by using standardized naming conventions

### PR DESCRIPTION
**Pull request by joonlim**

### CODE REVIEW

File: gke-deploy/README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ be used locally.
     ![deployed-resources](doc/deployed-resources.png)
 
     ```bash
-    curl 35.231.198.229
+    curl 35.196.67.253
```
Comment (joonlim) [inline gke-deploy/README.md]: I changed this because I updated image it is referring to^

**Comment by joonlim** [comment]
@stewchen please review

File: gke-deploy/deployer/deployer.go [hunk 2]
```diff
@@ -126,28 +116,25 @@ func (d *Deployer) Prepare(ctx context.Context, im name.Reference, appName, appV
 			return fmt.Errorf("failed to check if Namespace %q exists: %v", namespace, err)
 		}
 		if !ok {
-			fmt.Printf("Creating Namespace resource %q\n", namespace)
+			fmt.Printf("Creating Namespace object %q\n", namespace)
```
Comment (stewchen) [inline gke-deploy/deployer/deployer.go]: Would it be better to just say "Creating Kubernetes Namespace" or "Creating Namespace"

Comment (joonlim) [reply gke-deploy/deployer/deployer.go]: Added a change

**Comment by joonlim** [comment]
@ImJasonH please review/merge

**Pull request merged**